litex/build/altera/common.py

```diff
@@ -236,8 +236,8 @@ def __init__(self, i1, i2, o, clk):
                 p_asclr_ena = "ASCLR_ENA_NONE",
                 p_sclr_ena  = "SCLR_ENA_NONE",
                 o_dataout   = o[j],
-                i_datainlo  = i2[j],
-                i_datainhi  = i1[j],
+                i_datainlo  = i1[j], # rising edge
+                i_datainhi  = i2[j], # falling edge
```

Comment:
Look like this swap must be only applied to Agilex series: but this modification will changes behavior for all Altera FPGA's.
Maybe a specific Impl is required for Agilex?

Reply:
Yes true: sorry